lib/autocomplete-paths.js

```diff
@@ -39,6 +40,7 @@ export default {
     scopeOptions.forEach(scopeOption => {
       this.subscriptions.add(atom.config.observe(scopeOption, value => {
         if (!this._provider) return
+        this._provider.updateConfig()
```

Comment:
I don't think changing `autocomplete-paths.scopes` requires the `this._provider.updateConfig()` call.

Reply:
Fixed in https://github.com/atom-community/autocomplete-paths/pull/237/commits/ebbfea12d98dfa31dd166a36e909d95ac7870d41